src/components/dialog/dialog.stories.tsx

```diff
@@ -54,6 +59,11 @@ export const DefaultStory: StoryFn = () => {
           <Textbox label="Birth Place" />
           <Textbox label="Favourite Colour" />
           <Textbox label="Address" />
+          <TextEditor
+            onChange={handleEditorChange}
+            value={editorState}
+            labelText="Text Editor Label"
+          />
```

Comment:
I think that adding that to the story available for users would add more complexity to already complex example. What do you think of adding that to the test story instead?

Reply:
Yeah sounds like a better approach

Reply:
I agree with Kuba that we definitely shouldn't add this to the default story. Leave that as it is and add a new example in test stories with the Text Editor, as he suggests.